src/aap_eda/api/views/eda_credential.py

```diff
@@ -63,10 +57,16 @@ class EdaCredentialViewSet(
     queryset = models.EdaCredential.objects.all()
     serializer_class = serializers.EdaCredentialSerializer
     filter_backends = (
-        KindFilterBackend,
+        FieldLookupBackend,
         defaultfilters.DjangoFilterBackend,
     )
     filterset_class = filters.EdaCredentialFilter
+    rest_filters_reserved_names = [
+        "refs",
+        "force",
+        "name",
```

Comment:
@hsong-rh Why is name a reserved attribute, shouldn't customers be allowed to filter by name?

Reply:
no, ANSIBLE_BASE_REST_FILTERS_RESERVED_NAMES don't contain `name` and `credential_type_id`.